benchmark/cuchembm/metrics/sampling.py

```diff
@@ -143,16 +150,20 @@ def sample(self, smiles, num_samples, radius):
                                                       scaled_radius=radius,
                                                       force_unique=False,
                                                       sanitize=False)
+        generated_smiles = generated_smiles[1:]
+        if self.remove_invalid:
```

Comment:
@rilango Now removing invalid SMILES by default from uniqueness and novelty metrics